pytorch_lightning/core/datamodule.py

```diff
@@ -160,7 +161,13 @@ def has_prepared_data(self) -> bool:
 
         Returns:
             bool: True if ``datamodule.prepare_data()`` has been called. False by default.
+
+        .. deprecated::v1.4
```

Comment:
Not sure, but I thinks sphinx needs some spaces here...
```suggestion
        .. deprecated:: v1.4
```